docs/crd.md

```diff
@@ -269,9 +270,11 @@ FlinkCluster
         See [more info](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/)
     - **job** (optional): Job spec. If specified, the cluster is a Flink job cluster; otherwise, it is a Flink
       session cluster.
-      - **jarFile** (required): JAR file of the job. It could be a local file or remote URI, depending on which
-        protocols (e.g., `https://`, `gs://`) are supported by the Flink image.
-      - **className** (required): Fully qualified Java class name of the job.
+      - **jarFile** (optional): JAR file of the job. It could be a local file or remote URI, depending on which
+        protocols (e.g., `https://`, `gs://`) are supported by the Flink image. `jarFile` or `python` is required
+      - **className** (optional): Fully qualified Java class name of the job.
```

Comment:
className is optional but it is not yet updated.

https://github.com/spotify/flink-on-k8s-operator/blob/master/api/v1beta1/flinkcluster_types.go#L371